Makefile

```diff
@@ -23,7 +23,8 @@ CACHE := cache
 
 # Version of the source-controller from which to get the GitRepository CRD.
 # Pulls source-controller/api's version set in go.mod.
-SOURCE_VER ?= $(shell go list -m github.com/fluxcd/source-controller/api | awk '{print $$2}')
+# SOURCE_VER ?= $(shell go list -m github.com/fluxcd/source-controller/api | awk '{print $$2}')
+SOURCE_VER = 944f4cfa10c5
```

Comment:
Wait, this and the changes in go.mod should be reverted before merging, right?

Reply:
yes, this PR depends on a new release of `fluxcd/pkg/git` and SC